guides/common/modules/ref_capsule-ports-and-firewalls-requirements.adoc

```diff
@@ -77,13 +64,44 @@ This includes the base operating system on which a {SmartProxyServer} is running
 A DHCP {SmartProxy} performs ICMP ping and TCP echo connection attempts to hosts in subnets with DHCP IPAM set to find out if an IP address considered for use is free.
 This behavior can be turned off using `{foreman-installer} --foreman-proxy-dhcp-ping-free-ip=false`.
 
-.Optional Network Ports
-[cols="24%,20%,18%,38%a",options="header"]
+.{SmartProxy} outgoing traffic
+[cols="15%,15%,15%,15%,20%,20%",options="header"]
+
 |====
-| Port | Protocol | Service | Required For
-| 7911 | TCP | DHCP | * {SmartProxy} originated commands for orchestration of DHCP records (local or external).
-                      * If DHCP is provided by an external service, you must open the port on the external server.
+| Destination Port | Protocol | Service | Destination | Required For | Description
+| | ICMP | ping  | Client | DHCP | Free IP checking (optional)
+| 7 | TCP | echo | Client | DHCP |Free IP checking (optional)
+| 22 | TCP | SSH | Target host | Remote execution | Run jobs
+| 53 | TCP and UDP | DNS | DNS Servers on the Internet | DNS Server | Resolve DNS records (optional)
+| 53 | TCP and UDP | DNS | DNS Server | {SmartProxy} DNS | Validation of DNS conflicts (optional)
+| 68 | UDP | DHCP | Client | Dynamic IP | DHCP (optional)
+| 443 | TCP | HTTPS | {Project} | {SmartProxy} | {SmartProxy}
+
+Configuration management
+
+Template retrieval
+
+OpenSCAP
+
+Remote Execution result upload
```

Comment:
I am still NOT a fan of doing this. As discussed before, I would much rather prefer a solution that is easier to read, even if it means we have to duplicate port, protocoll, and service.

Reply:
What about a coma separated list? We can also drop Capsule from the description since it's already in "Required for".

Reply:
Folks have a tight schedule, let me merge this and we can take the discussion about improving the form somewhere else. Feel free to file a PR @maximiliankolb if you think you have a better idea.